content/version/2/0/code_of_conduct.da.md

```diff
@@ -0,0 +1,82 @@
+# Aftale om regler for deltagelse i fælleskabet
+
+## Vores løfte
+
+Som medlemmer og ledere lover vi at deltagelse i vores fællesskab bliver en chikanefri oplevelse for alle, uanset alder, kropsform, synlige eller usynlige handicap, etnicitet, køn, erfaringsniveau, uddannelse, social- eller økonomisk status, nationalitet, udseende, hudfarve, kaste, religion, eller seksuel orientering.
+
+Vi lover at deltage og samarbejde på en måde, der bidrager til at gøre fællesskabet åbent, imødekommende, mangfoldigt, inkluderende og velfungerende.
+
+## Vores standarder
+
+Eksempler på hvordan der skabes en god stemning i vores fællesskab inkluderer:
+
+* At vise medfølelse og være venlig overfor andre.
+* At have respekt for andre meninger, synspunkter, og erfaringer.
+* At give og kærligt at modtage konstruktiv feedback.
+* At tage ansvar og undskylde overfor dem, der bliver ramt af vores fejl og at lære af oplevelsen.
+* At fokusere på hvad der er bedst, ikke bare for os som individer, men for hele fællesskabet.
+
+Eksempler på uacceptabel opførsel inkluderer:
+
+* Brug af vulgært sprog eller vulgære billeder og seksuel opmærksomhed eller tilnærmelser af nogen art.
+* Ondsindet provokation, fornærmende eller nedladende kommentater, personlige eller politiske angreb.
+* Offentlig eller privat chikane.
+* Offentliggørelse af andres private information, f.eks. deling af deres e-mail eller postadresse uden deres tilladelse.
+* Anden opførsel som ikke hører til i et professionelt miljø.
+
+## Ansvar for håndhævelse
+
+Fællesskabets ledere er ansvarlige for at informere om og håndhæve vores standarder for acceptabel opførsel. Det sker ved på en retfærdig og proportionel måde at irettesætte opførsel som de bedømmer som uacceptabel, truende, offensiv eller skadelig.
+
```

Comment:
> Community leaders have the right and responsibility to remove, edit, or reject comments, commits, code, wiki edits, issues, and other contributions that are not aligned to this Code of Conduct, and will communicate reasons for moderation decisions when appropriate.

This part seems to be left out when comparing to the English version. Maybe it should be added as well?

Reply:
Ooooh, yes, this one is important! Should I make a new pull request, or can I make a review myself?

Reply:
@bilfri if you update the file in the `danish-v2_0` branch in your own fork, the changes should appear here:

https://github.com/bilfri/contributor_covenant/blob/danish-v2_0/content/version/2/0/code_of_conduct.da.md

You can either edit the file online through the GitHub website, or pull your branch down, add the missing section, and then push it back up.

Reply:
thx, I think it worked :) didn't know that changes committed after the pull request would be included - cool!